examples/triton/tensorflow/helpers.py

```diff
@@ -0,0 +1,409 @@
+from __future__ import annotations
+
+import os
+import json
+import time
+import typing as t
+from pathlib import Path
+
+import fs
+import attr
+import torch
+import fs.errors
+
+from bentoml._internal.utils import LazyLoader
+
+if t.TYPE_CHECKING:
+    import cv2
+    import numpy as np
+    import tensorflow as tf
+    import torchvision
+    from PIL import Image as PILImage
+    from fs.base import FS
+    from numpy.typing import NDArray
+    from torch.jit._script import ScriptModule
+
+    P = t.ParamSpec("P")
+
+else:
+    tf = LazyLoader("tf", globals(), "tensorflow")
+    np = LazyLoader("np", globals(), "numpy")
+    torchvision = LazyLoader("torchvision", globals(), "torchvision")
+    PILImage = LazyLoader("PILImage", globals(), "PIL.Image")
+    cv2 = LazyLoader("cv2", globals(), "cv2")
+
+WORKING_DIR = Path(__file__).parent
+MODEL_TYPE = os.environ.get("MODEL_TYPE", "yolov5s")
+MODEL_FILE = WORKING_DIR / f"{MODEL_TYPE}.pt"
+TORCH_DEVICE = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+
+
+def load_traced_script() -> tuple[ScriptModule, dict[str, t.Any]]:
```

Comment:
We don't need this in a tensorflow example? But I think for now it's easier to keep the same file for all 3 examples. Let's remove this at the final stage

Reply:
This is actually used in the train script.